### PR TITLE
css-four-column-layout - Junou Lee

### PR DESCRIPTION
**Pull request by junoulee**
image

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+  background-color: rgb(247 248 250);
+  margin-left: -40px;
+  margin-right: -40px;
+  padding-left: 100px;
+  padding-right: 100px;
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:16]: `row` should be `display: flex` and nothing else. Again, you need to review `css-layout-classes`.

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+.column-quarter {
+  padding-left: 10px;
+  padding-right: 10px;
+}
+
+.column-full {
+  flex-basis: 100%;
+  text-align: center;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:32]: `column` classes should have `flex-basis` applied to them, and nothing else.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-four-column-layout/styles.css (hunk 2 above)
Comment (junoulee) [reply css-four-column-layout/styles.css]: image

File: css-four-column-layout/index.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+      <div class="column-quarter">
+        <section>
+          <img class="bottom-image" src="./images/bars-icon.png" alt="bars-icon">
+          <h3>Stats + Mileage</h3>
+          <p>Tracks your trips across all apps.</p>
+        </section>
+      </div>
+    </div>
+
+  </div>
```
Comment (codingcodymiller) [inline css-four-column-layout/index.html:89]: I can understand why you would separate this into three rows based on setting it up with layout classes, but now your HTML makes it seem like the images, headers, and subtext are not related, and they absolutely are. You should instead have the images, header, and subtext together inside one `column-quarter`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-four-column-layout/index.html (hunk 3 above)
Comment (junoulee) [reply css-four-column-layout/index.html]: image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**